lc0/src/selfplay/game.cc

```diff
@@ -76,9 +76,16 @@ void SelfPlayGame::Play(int white_threads, int black_threads) {
 
 std::vector<Move> SelfPlayGame::GetMoves() const {
   std::vector<Move> moves;
+  int count = 0;
+  for (Node* node = tree_[0]->GetCurrentHead();
+	   node != tree_[0]->GetGameBeginNode(); node = node->GetParent()) {
+	++count;
+  }
+  bool flip = (count & 1) == 0;
```

Comment:
I think this will not work if game started with black's move (although I that's probably not possible currently for selfplay). Better, as we go backwards anyway:
```cpp
bool flip = !tree_[0]->IsBlackToMove();
```
Note that `!` is needed because after white's move position become's "is black to move".

if we went forward, it would be more complicated:
```cpp
bool flip = tree_[0]->GetPositionHistory().Starting().IsBlackToMove();
```

Reply:
changed, this is much better